tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+from OMPython import OMParser
```

Comment:
I definitely need to import from the `test_*.py` files for the tests to run (using `python -m unittest tests tests.test_OMParser` from the base directory). I'm not very used to working with packages (the current structure seems very baroque to me) but this is what I had to do to get the tests going. I'm open to improvement.